apps/condo/domains/marketplace/components/Invoice/InvoiceRowsTable.tsx

```diff
@@ -111,9 +112,9 @@ export const InvoiceRowsTable: React.FC<InvoiceRowsTableProps> = ({ invoice }) =
     const intl = useIntl()
     const ContractPriceMessage = intl.formatMessage({ id: 'pages.condo.marketplace.invoice.form.contractPrice' }).toLowerCase()
 
-    const currencyCode = get(invoice, 'context.currencyCode')
+    const currencyCode = get(invoice, 'currencyCode') || DEFAULT_INVOICE_CURRENCY_CODE
     const rows = useMemo(() => get(invoice, 'rows'), [invoice])
-    const organizationId = get(invoice, 'context.organization.id')
+    const organizationId = get(invoice, 'organization.id') || get(invoice, 'organization')
```

Comment:
why are you trying to get `organizationId` from `organization` if it should be in `organization.id`? It is ok?

Reply:
Yes, because there is can be invoice in ticket form (it's only front form data, organization will be in `invoice.organization` for pass organization in create invoice action after submit ticket form) or in existed invoice card/update form (there will be `invoice.organization.id`)